modules/ROOT/pages/agent-visualizer-components.adoc

```diff
@@ -0,0 +1,94 @@
+= Understand the Agent Visualizer Components
+
+Agent Visualizer shows the architectural structure of your agent network. Explore node types such as brokers, agents, and Model Context Protocol (MCP) servers, and edges that show declared and runtime relationships and traffic. Use layers and filters to focus your view, and detail cards to see metadata and metrics.
+
+[[node-types]]
+== Node Types
+
+There are different types of nodes in the agent network to provide a visual representation of different agentic assets (agents, brokers, and MCP servers) within the agent network. Nodes show you the structure, interactions, health, and governance of distributed agents, brokers, and MCP servers. 
+
+• Broker nodes
++
+Represent broker groups and enumerate the different broker agents they can communicate with. Broker groups are defined in the broker YAML and published to Anypoint Exchange.
+• Agents
++
+An agent is an autonomous software component that uses goals, context, and available tools (often via an LLM) to decide and execute actions on behalf of a user or system. The agent nodes include:
++
+** Agentforce agents
++
+Agents that originate from Agentforce. 
+** External agents
++
+Agents that originate from external, non-Salesforce platforms, such as OpenAI, Bedrock, Vertex AI, and so on. 
+** Brokers
++
+A broker is an intelligent routing service that coordinates task delegation across specialized agents in your enterprise. Brokers originate from MuleSoft and are referenced in the agent network YAML. 
+• MCP servers
++
+These nodes represent MCP servers. An MCP server is a service that implements the Model Context Protocol to expose tools and data to AI clients, enabling LLMs to invoke external capabilities through a standard interface.
```

Comment:
I wonder if it makes sense to have a glossary with these terms, so you don't interrupt the flow of this content with the definitions. It'd be easier to make the lists parallel in that case.